cdk/PgStacInfra.ts

```diff
@@ -57,6 +58,21 @@ export class PgStacInfra extends Stack {
     });
 
 
+    const buckets = ["nasa-maap-data-store"];
```

Comment:
Should this be from the configuration file?

Reply:
Yep that would be better.

Reply:
I think that my only comment is still just this configuration should be moved

Reply:
Thanks @jjfrench.

Just pushed a commit that addresses this.